examples/blockly-vue3/README.md

```diff
@@ -0,0 +1,35 @@
+# blockly-vue3
+
+This template should help get you started developing with Vue 3 in Vite.
+
+## Recommended IDE Setup
+
+[VSCode](https://code.visualstudio.com/) + [Volar](https://marketplace.visualstudio.com/items?itemName=Vue.volar) (and disable Vetur) + [TypeScript Vue Plugin (Volar)](https://marketplace.visualstudio.com/items?itemName=Vue.vscode-typescript-vue-plugin).
+
+## Customize configuration
+
+See [Vite Configuration Reference](https://vitejs.dev/config/).
+
+## Project Setup
+
+```sh
+npm install
+```
+
+### Compile and Hot-Reload for Development
+
+```sh
+npm run dev
+```
+
+### Compile and Minify for Production
+
+```sh
+npm run build
+```
+
+### Lint with [ESLint](https://eslint.org/)
+
+```sh
+npm run lint
+```
```

Comment:
Can you add a link to the [vuejs/art](https://github.com/vuejs/art) repo stating that that's where you acquired the logo.png ?

Reply:
Done. I didn't find a previous example of how to include it, if it's not in the right format don't hesitate to tell me :smile:

Reply:
That looks perfect :D Thank you so much!